freestyle-logging/jvm/src/main/scala/loggingJVM.scala

```diff
@@ -50,4 +52,50 @@ object loggingJVM {
           ME.catchNonFatal(logger.warn(msg, cause))
       }
   }
+
+  implicit def freeStyleLoggingKleisli[M[_], C: Manifest](
+      implicit ME: MonadError[M, Throwable]): FSHandler[LoggingM.Op, Kleisli[M, Logger, ?]] =
+    λ[FunctionK[LoggingM.Op, Kleisli[M, Logger, ?]]](_ match {
+      case LoggingM.DebugOP(msg) =>
+        Kleisli { logger =>
```

Comment:
Yeah, regarding wrapping everything in a single Kleisli, makes sense totally. I'm also trying to think of a better way to have this done. Kleisli might be a bit "foreign" for people using Freestyle. However, it can be done using the `Reader` effect (basically it's the same approach, but using already written effects). On the other hand, Freestyle is more inclined to implicits based injection, innit?

Regarding #319, yeah that'd be great! I can rebase #352 on top of this branch and keep it updated when we reach a decision on how to inject logger :)